Makefile

```diff
@@ -853,6 +853,11 @@ bench: bench-net bench-http bench-fs bench-tls
 
 bench-ci: bench
 
```

Comment:
That's right! I added it on [L717](https://github.com/watilde/node/blob/282455761a4a42ed659229f789ea7320baeb72de/Makefile#L717).